bin/node/runtime/src/lib.rs

```diff
@@ -377,6 +377,8 @@ impl pallet_democracy::Trait for Runtime {
 
 parameter_types! {
 	pub const CouncilMotionDuration: BlockNumber = 5 * DAYS;
+	pub const CouncilMaxMembers: pallet_collective::MemberCount = 100;
```

Comment:
there is no need for this. You should just use `DesiredMembers` value from elections module. These two are working together.

Reply:
okay based on the description, maybe it can be a different and higher value.